### PR TITLE
changes to two pop figs and captions

### PR DESCRIPTION
**Pull request by ckyriazis**
I made some minor changes to the two pop figs and captions in response to reviewer requests.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
LGTM, thanks @ckyriazis

File: popSimManu.tex [hunk 1]
```diff
@@ -937,7 +937,9 @@ \section*{Supplemental Figures}
 \caption{\textbf{Migration rate parameters estimated under a two-population \emph{Drosophila} model}.
 Here we show inferred migration rates from \dadi and \fastsimcoal.
 Data were generated by simulating replicate \emph{Drosophila} genomes under the \cite{li2006inferring} model and using the genetic map
-inferred in \cite{comeron2012many}.}
+inferred in \cite{comeron2012many}.
+Directional migration from Europe to Africa is represented as MIG_AF_EU and
+migration from Africa to Europe is represented as MIG_EU_AF.}
```
Comment (petrelharp) [inline popSimManu.tex:942]: Hm, this isn't quite right, is it? The figure shows $m_\text{AF-EU}$ and $m_\text{EU-AF}$, not MIG. And, we should say what the other things are - the Ns and T_DIV?  It is obvious what these are but it's also very easy just to say, for those to whom it's not obvious.

File: popSimManu.tex [hunk 2]
```diff
@@ -949,7 +951,9 @@ \section*{Supplemental Figures}
 Here we show inferred migration rates from \dadi and \fastsimcoal.
 Data were generated by simulating
 replicate human genomes under the \cite{gutenkunst2009inferring} model and using the genetic map
-inferred in \cite{international2007second}.}
+inferred in \cite{international2007second}.
+Directional migration from Europe to Africa is represented as MIG_AF_EU and
+migration from Africa to Europe is represented as MIG_EU_AF.}
```
Comment (petrelharp) [inline popSimManu.tex:956]: Same here.

**Comment by petrelharp** [comment]
Good start - but it doesn't quite match the figure - do you want to fix this up, @ckyriazis?

File: popSimManu.tex (hunk 1 above)
Comment (petrelharp) [reply popSimManu.tex:942]: never mind, sorry!

**Pull request merged**